common/presenters/allocation-to-summary-list-component.js

```diff
@@ -0,0 +1,40 @@
+const i18n = require('../../config/i18n')
+const filters = require('../../config/nunjucks/filters')
+
+function allocationToSummaryListComponent(allocation) {
+  const {
+    moves_count: movesCount,
```

Comment:
I think we should be careful using this. Abeer and I discussed removing this as it's not really necessary. The information can be inferred by the number of moves on an allocation.

We were also discussing passing a "count" as part of the request rather than an attribute within an actual allocation. So to future proof that we should use `allocation.moves.length` instead.

Reply:
Wait -- `moves_count` pre-exists to the allocations. How I understand it, is that `moves_count` represents the number of slots available, but the moves associated are only the already filled slots. Otherwise, how could we display the empty slots?
@cwrw can you confirm this?

Reply:
My understanding was the moves are always there, as we need to know them to be able to "fill" them at the later stages.

My understanding was that we would return the number of moves, and we can determine "unfilled" on the frontend as those that don't contain a `person` relationship. The `moves_count` is a superfluous piece of information that we can already determine.

Reply:
Discussed with @cwrw and we are keeping `moves_count` for the moment being. Eventually we will instead parse the moves and count which ones have a person assigned.

Reply:
Discussed again with @cwrw and @teneightfive and we may as well change it now, since `moves_count` will be deprecated at some point. Shall remove it.